scaleway/resource_k8s_cluster_beta.go

```diff
@@ -416,9 +493,11 @@ func resourceScalewayK8SClusterBetaRead(d *schema.ResourceData, m interface{}) e
 	d.Set("apiserver_url", response.ClusterURL)
 	d.Set("wildcard_dns", response.DNSWildcard)
 	d.Set("status", response.Status.String())
+	d.Set("upgrade_available", response.UpgradeAvailable)
 
 	// autoscaler_config
 	d.Set("autoscaler_config", []map[string]interface{}{clusterAutoscalerConfigFlatten(response)})
+	d.Set("auto_upgrade", []map[string]interface{}{clusterAutoUpgradeFlatten(response)})
```

Comment:
`clusterAutoUpgradeFlatten` should return a `[]map[string]interface{}`

Reply:
In taht case, so should clusterAutoscalerConfigFlatten?

Reply:
yes